### DISCUSSION TITLE
Artisan Temps Incorrect (Yoshan)

### DISCUSSION THREAD

**Discussion by gnosiscoffeeroasters** [artisan-roaster-scope/artisan#909] [Q&A] [answered]
My Artisan temps are not reading correctly. My ET reads 32F and my BT reads 472F. The temps do not change from these readings. Any ideas? I have a PC, roasting on a Yoshan, and I have already downloaded the Universal Windows Driver that Yoshan recommended.

![image](https://user-images.githubusercontent.com/108023043/175976232-18c7d717-4dd2-4c03-bc04-2cdbf47dab62.png)

**Comment by poundy** [comment, marked as answer]
The simplest answer here is that the configuration is wrong. 

You didn't mention what the number(s) on the roaster correspond to these values.  I suspect your roaster isn't using the same modbus register values as expected, and you will need to change those to have it put the right register value (roaster data point) into the right field in Artisan.

**Reply by gnosiscoffeeroasters** [reply]
I think what you said was the problem. I got on a remote access call with a Yoshan engineer, and he realized that the parameters were off. He had me change a number from 2 to 3 on the ET of my roaster control settings. It took him 40 minutes to figure it out. He might have also changed something in Artisan to fix the BT, but he was moving to fast for me to see exactly what he did.

**Reply by poundy** [reply]
For the benefit of others, if you could paste in an image of your modbus register map, that will mean we can see what differences there are and people can see it. 
This is the problem when roaster manufacturers change their hardware between revisions and don't share the details with the owner or software producers....
